templates/page/details.html

```diff
@@ -1,8 +1,24 @@
 {% extends "base.html" %}
 {% load i18n %}
 {% load materializecss %}
+{% load build_absolute_uri from urls %}
 
-{% block title %}{{ page.title }} - {{ block.super }}{% endblock %}
+{% block title %}
+  {% if page.seo_title %}
+    {{ page.seo_title }}
```

Comment:
Shouldn't we use `model.get_seo_title()` here? Then we wouldn't have to check if `seo_title` is set, as `get_seo_title` would always return what we want.

Reply:
What about a template tag? I feel that adding more methods to models goes against our recent changes that moved logic outside of the models.

Reply:
Logic will be moved to template tag, however in here there's quite different behaviour for `seo_title` and `title` - as below
```
{% block title %}
  {% if page.seo_title %}
    {{ page.seo_title }}
  {% else %}
    {{ page.title }} - {{ block.super }}
  {% endif %}
{% endblock %}
```
I dont want to join `seo_title` with `block.super` as it can exceed 70 chars (and be cut weirdly by Google)